### PR TITLE
Minor layout cleanup

### PR DESCRIPTION
**Pull request by mattpap**
Improves readability and removes bulky code.

### CODE REVIEW

File: bokehjs/src/coffee/core/layout/solver.coffee [hunk 1]
```diff
@@ -3,17 +3,10 @@ import {Variable, Expression, Constraint, Operator, Strength, Solver as Constrai
...
-    return new Constraint(expr, op)
+  (args) -> new Constraint(new Expression(args...), op)
 
 _weak_constrainer = (op) ->
-  () ->
-    args = [null]
-    for arg in arguments
-      args.push(arg)
-    new Constraint( new (Function.prototype.bind.apply(Expression, args)), op, Strength.weak )
+  (args) -> new Constraint(new Expression(args...), op, Strength.weak)
```
Comment (mattpap) [inline bokehjs/src/coffee/core/layout/solver.coffee]: @birdsarah, I presumptuously replaced this, but maybe there was a reason I overlook to write code like this?

Comment (bryevdv) [inline bokehjs/src/coffee/core/layout/solver.coffee]: I actually got this code from Chris Colbert IIRC, I thought there was some subtlety this was contending with but I can't recall any details. Maybe I was just unaware of `...` at the time, though.

Comment (birdsarah) [inline bokehjs/src/coffee/core/layout/solver.coffee]: If it works. I have no objection. I do recall some weirdness that required the bind.apply but I don't have anything specific, as @bryevdv said, I took the lead from him and @sccolbert.

Comment (birdsarah) [inline bokehjs/src/coffee/core/layout/solver.coffee]: Ultimately this just makes convenience functions around solver so it should be pretty easy to know if this didn't work.

**Comment by mattpap** [comment]
This doesn't work. I will revisit this at some point.

**Pull request closed without merging**